docs/sync.js

```diff
@@ -40,7 +40,8 @@
  * @property {Realm.User} user - A {@link Realm.User} object obtained by calling `Realm.App.logIn`.
  * @property {string|number|BSON.ObjectId|null} partitionValue - The value of the partition key.
  * @property {function} [error] - A callback function which is called in error situations.
- *    The `error` callback can take up to five optional arguments: `name`, `message`, `isFatal`,
+ *    The errpr callback can take up to two arguments: `session` and `syncErr`. If `syncError.name == "ClientReset"`, `syncError.path` and `syncError.config` are set
+ *    and `syncError.readOnly` is true. Otherwise, `syncError` can have up to five properties: `name`, `message`, `isFatal`,
  *    `category`, and `code`.
```

Comment:
Wording suggestion (incl. typos):
```suggestion
 *    The error callback is called with two arguments: `session` and `syncError`.
 *    If `syncError.name === "ClientReset"`, `syncError` will contain the properties `path`, `config` & `readOnly === true`.
 *    Otherwise, `syncError` contains the properties: `name`, `message`, `isFatal`, `code` & `category` (`category` is optional).
```

(Not sure if this suggestion uphold the char-length, it's a bit hard to see in a gh-suggestion)

Reply:
The generated HTML seems file